content/src/storage/FileSystemContentStorage.ts

```diff
@@ -84,7 +84,7 @@ export class FileSystemContentStorage implements ContentStorage {
     return (await this.stats(id))?.size
   }
 
-  async fixContentItem(id: string, oldContentPath: string): Promise<void> {
+  async storeExistingContentItem(id: string, oldContentPath: string): Promise<void> {
```

Comment:
Please be more specific
```suggestion
  async storeExistingContentItem(fileInDisk: string, contentHash: string): Promise<void> {
```